src/pages/resultsView/mutation/ResultsViewMutationMapper.tsx

```diff
@@ -65,6 +65,9 @@ export default class ResultsViewMutationMapper extends MutationMapper<IResultsVi
 
     protected mutationTableComponent(): JSX.Element|null
     {
+        const canonicalTranscriptId = this.props.store.canonicalTranscript.result && this.props.store.canonicalTranscript.result.transcriptId;
+        const transcript = this.props.store.activeTranscript && (this.props.store.activeTranscript === canonicalTranscriptId)? this.props.store.canonicalTranscript.result : this.props.store.transcriptsByTranscriptId[this.props.store.activeTranscript!];
```

Comment:
This line is really long, it's a good practice to divide it into multiple lines. Maybe something like this:

```suggestion
        const transcript = this.props.store.activeTranscript && (this.props.store.activeTranscript === canonicalTranscriptId) ?
            this.props.store.canonicalTranscript.result : this.props.store.transcriptsByTranscriptId[this.props.store.activeTranscript!];
```